src/test/java/com/integrivideo/stories/ManageComponentStoryTest.java

```diff
@@ -50,6 +51,21 @@ public void componentCanBeUpdated() {
         componentSteps.componentDetailsShouldBeLike(ComponentTypeEnum.CLOUD_VIDEO_RECORDER, "edited");
     }
 
-    //TODO check prices for each component
-
+    @Test
+    public void componentPrices() {
+        projectSteps.createProject();
+        projectSteps.openProjectPage(projectSteps.getProjectCount() - 2);
+        componentSteps.openCreateComponentPage();
+        componentSteps.createComponent(ComponentTypeEnum.VIDEO_CHAT, "name");
+        componentSteps.returnToProject();
+        projectSteps.projectPricesShouldBeLike(Data.PRICES_VC);
+        componentSteps.openCreateComponentPage();
+        componentSteps.createComponent(ComponentTypeEnum.CLOUD_VIDEO_RECORDER, "name");
+        componentSteps.returnToProject();
+        projectSteps.projectPricesShouldBeLike(Data.PRICES_VC_CVR);
+        componentSteps.openCreateComponentPage();
+        componentSteps.createComponent(ComponentTypeEnum.MULTI_DEVICE_VIDEO_PLAYER, "name");
+        componentSteps.returnToProject();
+        projectSteps.projectPricesShouldBeLike(Data.PRICES_VC_CVR_MVP);
```

Comment:
As said in my previous comment let's do something like 

projectSteps.projectPricesShouldBeLike(10, 0, 10);